_data/events.yml

```diff
@@ -215,6 +205,39 @@
   githubUrl: ""
   blogPost: ""
 
+- name: "BBQ en Boardgames Bonanza™"
+  date: 2024-08-30
+  hide: true
+  track: "FunEvent"
+  summary: >
+    BBQ + Boardgame Bonanza with Thomas Janssens was a hit! 
+    Itenium colleagues came together for an evening of grilling 
+    and board gaming. Games were competitive, and the atmosphere 
+    was relaxed, creating a fun break from the usual work routine. 
+    Whether it was rolling dice or flipping cards, everyone had a 
+    great time.
+
+  level: Introduction
+  content: "Hands-On"
+  type: ["On-Site"]
+  duration: "2h"
+  image: "assets/images-events/BBQ-and-Boardgame-Bonanza-1.jpg"
+  gallery:
+    - img: "assets/images-events/BBQ-and-Boardgame-Bonanza-2.jpg"
+      text: "Team members strategize over a board game, blending IT fun and teamwork."
+    - img: "assets/images-events/BBQ-and-Boardgame-Bonanza-3.jpg"
+      text: "Colleagues engage in a competitive board game, mixing strategy and IT camaraderie."
+    - img: "assets/images-events/BBQ-and-Boardgame-Bonanza-4.jpg"
+      text: "Thomas shows off the BBQ with a peace sign, adding a fun touch to the IT event."
+    - img: "assets/images-events/BBQ-and-Boardgame-Bonanza-5.jpg"
+      text: "Board game action under evening lights, capturing IT team fun and strategy."
+    - img: "assets/images-events/BBQ-and-Boardgame-Bonanza-6.jpg"
+      text: "The team shares a meal, enjoying food and conversations in a relaxed IT setting."
```

Comment:
The pictures on the social media are much nicer, I would definitely re-use some of those
On our site, there can be more pictures compared to the socials, especially for a FunEvent like this one